dietpi/dietpi-software

```diff
@@ -6427,6 +6435,52 @@ _EOF_
 			Download_Test_Media
 		fi
 
+		if To_Install 212 kavita # Kavita
+		then
+			case $G_HW_ARCH in
+				1) local arch='arm';;
+				2) local arch='arm';;
+				3) local arch='arm64';;
+				*) local arch='x64';;
+			esac
+
+			local fallback_url="https://github.com/Kareadita/Kavita/releases/download/v0.7.8/kavita-linux-$arch.tar.gz"
+			Download_Install "$(curl -sSfL 'https://api.github.com/repos/Kareadita/Kavita/releases/latest' | mawk -F\" "/\"browser_download_url\": .*\/kavita-linux-$arch\.tar\.gz\"/{print \$4}")" /opt
+
+			# User
+			Create_User -g dietpi -d /mnt/dietpi_userdata/kavita kavita
+
+			# Data
+			G_EXEC mkdir -p /mnt/dietpi_userdata/{ebooks,comics}
+
+			# Service # https://wiki.kavitareader.com/en/install/linux-install
+			cat << _EOF_ > /etc/systemd/system/kavita.service
+[Unit]
+Description=Kavita Server (DietPi)
+Wants=network-online.target
+After=network-online.target remote-fs.target
```

Comment:
```suggestion
After=network-online.target remote-fs.target
StartLimitIntervalSec=60
StartLimitBurst=5
```
I'm not a big fan of `Restart=on-failure`. If a service crashes, then in 95% of the cases a restart will just lead to another crash and repeating it makes things only worse + masks the relevant logs with the repeated restart attempts. Usually if there is a crash, then admins should investigate and fix the underlying reason instead of having the service in an unlimited restart loop. However, if this is seen as necessary here, then at least there must be a limit: max 5 restarts in 60 seconds.